### PR TITLE
return index of repository. part of #33

### PR DESCRIPTION
**Pull request by artemlazarev**
Part of work for implementing #33
Initial class and responses  for generating responce acording https://www.python.org/dev/peps/pep-0503/ specification.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#39](https://github.com/artipie/pypi-adapter/issues/39) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/695e1879-d45f-4cb4-a3a7-f4cd6bdba366); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: pom.xml [hunk 1]
```diff
@@ -134,6 +134,14 @@ SOFTWARE.
           <trimStackTrace>false</trimStackTrace>
         </configuration>
       </plugin>
+        <plugin>
```
[the remark this answers is not in the document]
Comment (artemlazarev) [reply pom.xml]: @g4s8 strange. i had removed this unnecessary dependency 7 hours ago. But you had seen and commented it 3 hours ago. gone out.

File: src/main/java/com/artipie/pypi/HtmlIndexResponse.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import com.artipie.http.rs.RsWithStatus;
+import java.nio.ByteBuffer;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * HtmlIndexResponse produce and format html index of the pypi repository.
+ *
+ * @since 0.2
+ */
+final class HtmlIndexResponse  implements Response {
```
[the remark this answers is not in the document]
Comment (artemlazarev) [reply src/main/java/com/artipie/pypi/HtmlIndexResponse.java:46]: @g4s8  made redesign. And add puzzle.

**Comment by g4s8** [comment]
@artemlazarev please check build errors: https://github.com/artipie/pypi-adapter/pull/39/checks?check_run_id=783209813

**Comment by 0crat** [comment]
Job was finished in 24 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**